### ISSUE TITLE
Random FPS drops

### ISSUE THREAD

**Issue by JQINX** [ValveSoftware/Dota2-Gameplay#9710]
### Description

I don't know if anyone else is experiencing this or not the Fps are randomly dropping in game. I think some recent updates released has caused this. Checked in lobby and can confirm its not the driver but the game as I tried new and old driver. Game was running fine few weeks ago.

**Comment by Aluerie** [comment]
#9085 upvote/comment/search

**Comment by JQINX** [comment]
Yes I checked those links before posting and it seems problem hasn't been fixed yet. Waiting for a fix on fps at least.
